apis/fluent-en_GB/atrium-api-fluent-en_GB-common/src/main/kotlin/ch/tutteli/atrium/api/fluent/en_GB/anyAssertions.kt

```diff
@@ -41,6 +41,19 @@ fun <T> Expect<T>.isSameAs(expected: T): Expect<T> = _logicAppend { isSameAs(exp
  */
 fun <T> Expect<T>.isNotSameAs(expected: T): Expect<T> = _logicAppend { isNotSameAs(expected) }
 
+/**
+ * Allows to state a reason for one or multiple assertions for a given subject.
```

Comment:
The reason will be for the *current* subject, not for some provided one

```suggestion
 * Allows to state a reason for one or multiple assertions for the current subject.
```